# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webgpu/RenderPassEncoder.h
# Commit: e98d0558c0bf
# Full Hash: e98d0558c0bf2c4e145c07c4c8da145bbde6f05b
# Author: Brad Werth <bwerth@mozilla.com>
# Date: 2024-04-23 21:41:25
# Regressor Bug: 1712963
# File Overlap Count: 3
# Description:
#   Bug 1712963: Simplify mValid handling in some WebGPU classes. r=webgpu-reviewers,nical
#   
#   Primarily this standardizes ComputePassEncoder, RenderBundleEncoder and
#   RenderPassEncoder on the "if (!mValid)" pattern for early exits for ffi
#   functions. It also attempts to make each class have at most two ways to
# ==============================================================================

diff -r d1cde4aa5f35 -r e98d0558c0bf dom/webgpu/RenderPassEncoder.h
--- a/dom/webgpu/RenderPassEncoder.h	Tue Apr 23 15:21:21 2024 +0000
+++ b/dom/webgpu/RenderPassEncoder.h	Tue Apr 23 15:28:22 2024 +0000
@@ -49,7 +49,7 @@
 
  protected:
   virtual ~RenderPassEncoder();
-  void Cleanup() {}
+  void Cleanup();
 
   std::unique_ptr<ffi::WGPURecordedRenderPass, ffiWGPURenderPassDeleter> mPass;
   // keep all the used objects alive while the pass is recorded
